packages/vm/tests/api/state/stateManager.js

```diff
@@ -129,47 +130,35 @@ tape('StateManager', (t) => {
       st.end()
     },
   )
+  t.test('should generate the genesis state root correctly for mainnet from ethereum/tests data', async st => {
+    const genesisData = require('ethereumjs-testing').getSingleFile(
+      'BasicTests/genesishashestest.json',
+    )
+    const stateManager = new StateManager()
+
+    await stateManager.generateCanonicalGenesis()
+    let stateRoot = await stateManager.getStateRoot()
+    st.equals(
+      stateRoot.toString('hex'),
+      genesisData.genesis_state_root,
+      'generateCanonicalGenesis should produce correct state root for mainnet from ethereum/tests data',
+    )
+    st.end()
+  })
 
-  t.test('should generate the genesis state root correctly for mainnet', async (st) => {
-    if (isRunningInKarma()) {
-      st.skip('skip slow test when running in karma')
-      return st.end()
-    }
```

Comment:
it looks like the ci check failure is due to these lines being removed (it times out)

Reply:
I have no idea why I deleted this 🤔

Reply:
Could've been on my rebase. It was a nasty one.